var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -525,6 +525,9 @@ def cmake_args(self):
         if target >= "zen4":
             # AMD Family 17h (EPYC Genoa)
             options.append("-DGMX_SIMD=AVX_512")
+        elif target >= "zen3":
+            # AMD Family 17h (EPYC Milan)
+            options.append("-DGMX_SIMD=AVX_256")
```

Comment:
This is not needed, `zen3` is greater than `zen2` and the flag is the same

Reply:
Oh, apologies, I misread. Any reason why we use `AVX_256` instead of `AVX2_256`?

Reply:
ping @pelahi

Reply:
Sorry, fixed the typo.

Reply:
Ok, then I go back at https://github.com/spack/spack/pull/39332#discussion_r1286785800 This PR is not needed, since `> zen 3` is caught by `> zen2`